packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -776,8 +776,8 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   Rect describeApproximatePaintClip(RenderObject child) => _overflow > 0.0 ? Offset.zero & size : null;
 
   @override
-  String toString() {
-    String header = super.toString();
+  String toShortDescription() {
```

Comment:
For widgets it's often helpful. For render objects it's usually more helpful to know the identity of the render object than its precise state at the time of printing, though obviously both are sometimes useful (hence toStringShallow).